project/Cargo.toml

```diff
@@ -10,4 +10,4 @@ aead = { version = "0.5" }
 hkdf = { version = "0.12" }
 sha2 = { version = "0.10" }
 getrandom = { version = "0.2" }
-hmac = { version = "0.12", features = ["reset"]}
\ No newline at end of file
+hmac = { version = "0.12"}
```

Comment:
BTW if you have only version field, then you can write it simply as `hmac = "0.12"`.